core/src/Kokkos_Atomics_Desul_Wrapper.hpp

```diff
@@ -0,0 +1,239 @@
+#ifndef KOKKOS_DESUL_ATOMICS_WRAPPER_HPP_
+#define KOKKOS_DESUL_ATOMICS_WRAPPER_HPP_
+#include <Kokkos_Macros.hpp>
+
+#ifdef KOKKOS_ENABLE_IMPL_DESUL_ATOMICS
+#include <desul/atomics.hpp>
+
+#include <impl/Kokkos_Atomic_Memory_Order.hpp>
+#include <impl/Kokkos_Volatile_Load.hpp>
+
+// clang-format off
```

Comment:
I wouldn't do it just here

Reply:
?

Reply:
https://github.com/kokkos/kokkos/pull/3247/files#r633993453